app/views/users/profile.html.erb

```diff
@@ -70,7 +70,7 @@
 
     <div class="tab-pane" id="questions">
       <div class="btn-group text-center">
-        <a href="#asked" class="btn btn-default active" data-toggle="tab">Asked</a>
+        <a href="#asked" class="btn btn-default " data-toggle="tab">Asked</a>
```

Comment:
Great work @ibnjunaid !
But, can you please remove an extra space after ```btn-default```
```suggestion
        <a href="#asked" class="btn btn-default" data-toggle="tab">Asked</a>
```

Reply:
Yes.Of course. Thank you for review.

Reply:
@divyabaid16 Done .Please review.